components-api/src/main/scala/pl/touk/nussknacker/engine/api/json/encoders/TypeEncoders.scala

```diff
@@ -0,0 +1,87 @@
+package pl.touk.nussknacker.engine.api.json.encoders
+
+import io.circe._
+import io.circe.Json._
+import pl.touk.nussknacker.engine.api.json.TypingType
+import pl.touk.nussknacker.engine.api.json.TypingType.typeField
+import pl.touk.nussknacker.engine.api.typed.typing
+import pl.touk.nussknacker.engine.api.typed.typing._
+import pl.touk.nussknacker.engine.util.Implicits.RichScalaMap
+
+//TODO: refactor way of encoding to easier handle decoding.
+
+// If changes are made to Encoders/Decoders should also change Schemas in NodesApiEndpoints.TypingDtoSchemas for OpenApi
+object TypeEncoders {
```

Comment:
Moved from another file, which previosuly contained both encoders and decoders, without changes in code.